dist/robots.cfg

```diff
@@ -201,6 +215,14 @@ RobotIP <<EOR
     82.99.30.71,	# Munax
     82.99.30.72,	# Munax
     82.99.30.73,	# Munax
+    185.54.12*, # AlertLogic
```

Comment:
This looks like it's missing a `.`, should be `185.54.12.*` I would think.

Reply:
added.